docs/guides/compute-freeform.mdx

```diff
@@ -42,6 +42,32 @@ origin|destinations|percentile|cutoff|access
 2|Total Jobs|5|30|142600
 etc
 
+## Dual Accessibility
+The time needed to reach a threshold number of destinations (such as a health center or three grocery stores), also known as ("dual" accessibility)[https://doi.org/10.1111/gean.12220], can be calculated with an experimental feature in Conveyal.
+
+This feature is not yet available through the regional analysis creation interface. To activate it, add the following to your (custom analysis request JSON)[analysis/configuration#additional-settings].
+
+```JSON
+"includeTemporalDensity": true,
+"dualAccessibilityThreshold": n
+```
+where n is the desired threshold number of opportunities.
+
+The resulting CSV file will have the following columns:
+* origin: Id of the origin point
+* destinations: Id of the destination pointset
+* percentile: travel time percentile
+* D: the number of minutes required to reach a threshold of n opportunities. A value of -1 indicates the specified threshold is not reached within the maximum cutoff specified when the regional analysis was created.
+* m [0...119]: the number of opportunities reached in a given minute of travel time (not cumulative). For example values in the column 10 will be the number of opportunities reached between 10 and 11 minutes.
+
+### Example results
+
+The example below shows dual accessibility analysis results for access to health centers, where the `dualAccessibilityThreshold` has been set to 2. From a specific origin, given 5th percentile travel times, it takes 2 minutes to reach a total of 2 health centers. Given 25th percentile travel times, it takes 3 minutes to reach a total of 2 health centers.
+
+origin|destinations|percentile|D|0|1|2|3|...|118|119
```

Comment:
```suggestion
origin|destinations|percentile|dual|0|1|2|3|...|118|119
```

Reply:
Should we also explain the one-minute bins? They might wonder what those remaining 120 columns are. "The dual accessibility is derived from the data in the remaining columns: the marginal number of facilities reached with each one-minute increase in travel time. For example, the column 10 shows how many additional facilities are reached between ten and eleven minutes of travel."

Reply:
> Should we also explain the one-minute bins? They might wonder what those remaining 120 columns are. "The dual accessibility is derived from the data in the remaining columns: the marginal number of facilities reached with each one-minute increase in travel time. For example, the column 10 shows how many additional facilities are reached between ten and eleven minutes of travel."

This is explained a few lines above. I'll repeat some of the explanation here, but not duplicate the column 10 example.